modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -1068,42 +943,32 @@ test(
 	},
 
 	async ({apiHelpers, page}) => {
-		const pageTitle1 = 'My Page 1';
-		const sitePage1 = await createSitePage({
-			apiHelpers,
-			pageTitle: pageTitle1,
-		});
-		const pageTitle2 = 'My Page 2';
-		const sitePage2 = await createSitePage({
-			apiHelpers,
-			pageTitle: pageTitle2,
-		});
+		const pageName = 'My Page 2';
 
-		const channelName = 'My Property - ' + getRandomString();
-		await test.step('Connect the DXP to AC', async () => {
-			await syncAnalyticsCloud({
-				apiHelpers,
-				channelName,
-				page,
-			});
+		await createSitePage({
+			apiHelpers,
+			pageTitle: pageName,
+			siteName,
 		});
 
 		await test.step('Go to My Page 1', async () => {
 			await navigateToSitePage({
 				page,
-				pageName: pageTitle1,
+				pageName: pageTitle,
```

Comment:
@rafaellajordao, it would be much effort to change this `pageName` attribute to be `pageTitle`, if it would, we can do that in another pull request :smile:

Reply:
i'll open a ticket to work on it! [LPD-36814](https://liferay.atlassian.net/browse/LPD-36814)